### ISSUE TITLE
Consider indexing runtime specific symbols based on compiler version

### ISSUE THREAD

**Issue by roman01la** [snoe/clojure-lsp#64]
I'm thinking now that a set of symbols can be different across compiler versions, while the core vars are staying the same, there are also might be additions over time. Not that it's really important, but might be useful. Although this would be not as trivial task as including symbols directly into sources.

**Comment by snoe** [comment]
Yes, if people on old versions start getting annoyed by this we should add it, upvote the issue if it's important to you.

However my feeling about our list of symbols in clojure-core is that they should be somewhat unnecessary, they existed before I was fully parsing clojure.core and it ought to be possible to now pull 99% of the symbols out programmatically based on which jar is on your path, I just haven't had the time to dive in there.
